packages/agent/src/app.ts

```diff
@@ -156,12 +157,38 @@ export class App {
           // @ts-expect-error - Deprecated message type
           case 'transmit':
           case 'agent:transmit:response':
-            this.addToHl7Queue(command);
+            if (this.config?.status !== 'active') {
+              const err = new Error('Agent.status is currently set to off');
```

Comment:
Rather than creating a `new Error()` you can just use the string in both `this.log.error()` and `body:`